web-extension/search.js

```diff
@@ -121,26 +127,39 @@ function _onSearchResults(response, isHostQuery) {
     searching = false;
 }
 
-function _onResultSelected(event, element) {
-    element = element || event.target;
-    const value = element.innerText;
+function _onEntryAction(event, element) {
+    if (element) {
+        event.target = element;
```

Comment:
This won't work. `event.target` is a getter [according to the DOM spec](https://dom.spec.whatwg.org/#dom-event-target). Setting it is silently ignored in Chrome and throws and error in Firefox: `TypeError: setting getter-only property "target"`.